pkg/stages/steps_init.go

```diff
@@ -49,10 +49,10 @@ func GetInitrdStage(sys values.System, logger types.KairosLogger) ([]schema.Stag
 			return []schema.Stage{}, err
 		}
 
-		dracutCmd := fmt.Sprintf("dracut -f /boot/initrd %s", kernel)
+		dracutCmd := fmt.Sprintf("dracut -f /boot/initrd %s --no-hostonly", kernel)
 
 		if logger.GetLevel() == 0 {
-			dracutCmd = fmt.Sprintf("dracut -v -f /boot/initrd %s", kernel)
+			dracutCmd = fmt.Sprintf("dracut -v -f /boot/initrd %s --no-hostonly", kernel)
```

Comment:
adding the `--no-hostonly` flag in 25.05 fixes it, no clue why it doesn't pick the config file properly. Maybe this is not the right way to solve it but leaving it for now so we can start the conversation

Reply:
yeah good point, how could I assess that?

Reply:
it looks ok, the config comes from `/etc/dracut.conf.d/10-immucore.conf` so Im not sure if there is maybe another config file that comes afterwards and overrides that value??